SCAFFOLDING.md

```diff
@@ -14,7 +14,11 @@ Before writing a `webpack-cli` scaffold, think about what you're trying to achie
 
 ## webpack-addons-yourpackage
 
-In order for `webpack-cli` to compile your package, it relies on a prefix of `webpack-addons`. The package must also be published on npm. If you are curious about how you can create your very own `addon`, please read [How do I compose a webpack-addon?](https://github.com/ev1stensberg/webpack-addons-demo).
+In order for `webpack-cli` to compile your package, it must be available on npm or on your local filesystem.
+
+If the package is on npm, its name must have a prefix of `webpack-addons`.
+
+If the package is on your local filesystem, it can be named whatever you want. Pass the name of the package as a relative path to its root directory.
```

Comment:
i just noticed that you've removed `[How do I compose a webpack-addon?](https://github.com/ev1stensberg/webpack-addons-demo).` I would put it back was a good reference to me some time ago.

Reply:
Good catch, I'll restore that.